### PR TITLE
(typo) in install.sh

### PR DESCRIPTION
**Pull request by Geno1024**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ download_safeu_cli() {
 }
 
 install_scope() {
-    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Defualt: Y]: "
+    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Default: Y]: "
```
Comment (Triple-Z) [inline install.sh:59]: What’s the difference?

**Comment by Triple-Z** [comment]
@Geno1024 and, don’t merge to `master`, using `dev` instead.

**Comment by Geno1024** [comment]
A small typo at the end of this prompt. Def**au**lt vs Def**ua**lt. Though there is a common way of such an expression, for example [Y/n], or [YES/no/ignore], which the uppercase is the default choice.
&nbsp;

Geno Whirl!  暨南大学 15 级国际学院 CST（计算机科学与技术）陈毅钊 提交 


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Albert"<notifications@github.com&gt;; 
发送时间: 2020年6月9日(星期二) 晚上11:58
收件人: "NUAA-Open-Source/safeu-cli"<safeu-cli@noreply.github.com&gt;; 
抄送: "Geno"<754097987@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [NUAA-Open-Source/safeu-cli] (typo) in install.sh (#8)





 
@Triple-Z commented on this pull request.
 
 
In install.sh:
 &gt; @@ -56,7 +56,7 @@ download_safeu_cli() {  }    install_scope() { -    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Defualt: Y]: " +    printf "${YELLOW}Install safeu command tool globally (require sudo permission later) ? [Y/N, Default: Y]: "  
What’s the difference?

File: install.sh (hunk 1 above)
Comment (Geno1024) [reply install.sh:59]: A typo at the end of this prompt, def**au**lt vs def**ua**lt.

Though, there is a common way for expressing such choice, for example `[Y/n]`, `[no/YES/skip]`, where the UPPERCASE is default choice.

Comment (Triple-Z) [reply install.sh:59]: Okay, if I use the `[Y/N]`, how to represent the default value? Or using a infinite input loop no stop until read `Y` or `N`?

**Comment by Triple-Z** [comment]
Consider we will use args to represent this read message, this pr is no longer needed. Thanks for your contribution.

**Pull request closed without merging**